pkgs/applications/video/frigate/default.nix

```diff
@@ -85,31 +93,35 @@ python.pkgs.buildPythonApplication rec {
   postPatch = ''
     echo 'VERSION = "${version}"' > frigate/version.py
 
+    substituteInPlace frigate/events/audio.py \
+      --replace-warn "/cpu_audio_model.tflite" "${audio_model}" \
+      --replace-warn "/audio-labelmap.txt" "${placeholder "out"}/share/frigate/audio-labelmap.txt"
+
     substituteInPlace frigate/app.py \
-      --replace "Router(migrate_db)" 'Router(migrate_db, "${placeholder "out"}/share/frigate/migrations")'
+      --replace-warn "Router(migrate_db)" 'Router(migrate_db, "${placeholder "out"}/share/frigate/migrations")'
 
     substituteInPlace frigate/const.py \
-      --replace "/media/frigate" "/var/lib/frigate" \
-      --replace "/tmp/cache" "/var/cache/frigate" \
-      --replace "/config" "/var/lib/frigate" \
-      --replace "{CONFIG_DIR}/model_cache" "/var/cache/frigate/model_cache"
+      --replace-warn "/media/frigate" "/var/lib/frigate" \
+      --replace-warn "/tmp/cache" "/var/cache/frigate" \
+      --replace-warn "/config" "/var/lib/frigate" \
+      --replace-warn "{CONFIG_DIR}/model_cache" "/var/cache/frigate/model_cache"
 
     substituteInPlace frigate/http.py \
-      --replace "/opt/frigate" "${placeholder "out"}/${python.sitePackages}"
+      --replace-warn "/opt/frigate" "${placeholder "out"}/${python.sitePackages}"
 
     substituteInPlace frigate/output.py \
-      --replace "/opt/frigate" "${placeholder "out"}/${python.sitePackages}"
+      --replace-warn "/opt/frigate" "${placeholder "out"}/${python.sitePackages}"
 
     substituteInPlace frigate/detectors/detector_config.py \
-      --replace "/labelmap.txt" "${placeholder "out"}/share/frigate/labelmap.txt"
+      --replace-warn "/labelmap.txt" "${placeholder "out"}/share/frigate/labelmap.txt"
 
     substituteInPlace frigate/config.py \
-      --replace "/cpu_model.tflite" "${tflite_cpu_model}" \
-      --replace "/edgetpu_model.tflite" "${tflite_edgetpu_model}"
+      --replace-warn "/cpu_model.tflite" "${tflite_cpu_model}" \
+      --replace-warn "/edgetpu_model.tflite" "${tflite_edgetpu_model}"
 
     substituteInPlace frigate/test/test_config.py \
-      --replace "(MODEL_CACHE_DIR" "('/build/model_cache'" \
-      --replace "/config/model_cache" "/build/model_cache"
+      --replace-warn "(MODEL_CACHE_DIR" "('/build/model_cache'" \
+      --replace-warn "/config/model_cache" "/build/model_cache"
```

Comment:
```diff
- --replace-warn
+ --replace-error
```
Warnings wouldn't prevent automatic package updates from breaking Frigate's functionality in unexpected ways.